src/Magnum/Math/DualQuaternion.h

```diff
@@ -246,6 +246,19 @@ template<class T> class DualQuaternion: public Dual<Quaternion<T>> {
             return {q, Quaternion<T>(matrix.translation()/2)*q};
         }
 
+        /**
+         * @brief Create dual quaternion from rotation quaternion and translation vector
+         *
+         * @f[
+         *      \hat q = p + \epsilon p \cdot [\frac{\boldsymbol v}{2}, 0]
+         * @f]
+         *
+         * @see @ref translation(), @ref rotation()
+         */
+        static DualQuaternion<T> from(const Quaternion<T>& rotation, const Vector3<T>& translation) {
```

Comment:
This basically mirrors the API of [Matrix4::from(const Matrix3x3<T>&, const Vector3<T>&)](https://doc.magnum.graphics/magnum/classMagnum_1_1Math_1_1Matrix4.html#ab2c680b969e7f8f048f3dac2cb27c810), right? Would be great to cross-reference those two in the docs (and then do the same for `Matrix3` and `DualComplex` so 2D isn't left behind).

Reply:
Matrix3::from already exists, I added the cross references to the others there.
Adding DualComplex::from (that really has a 2D transform equivalent like quaternions in 3D?)